tools/ci_testing/addons_gpu.sh

```diff
@@ -39,12 +39,18 @@ ls -alh $PYTHON_BIN_PATH
 # Use default configuration here.
 yes 'y' | ./configure.sh
 
+wget https://raw.githubusercontent.com/tensorflow/tensorflow/master/tensorflow/tools/ci_build/gpu_build/parallel_gpu_execute.sh
```

Comment:
I think it's a bit more controlled if we copy this script into our own repo? WDYT?

Reply:
Sure, I also want to copy it into the repo :-)